src/main/kotlin/uk/gov/justice/digital/hmpps/incentivesapi/service/PrisonerIepLevelReviewService.kt

```diff
@@ -76,15 +76,20 @@ class PrisonerIepLevelReviewService(
   }
 
   @Transactional
-  suspend fun persistPrisonerIepLevel(bookingId: Long, syncPostRequest: SyncPostRequest): IepDetail {
+  suspend fun persistSyncPostRequest(bookingId: Long, syncPostRequest: SyncPostRequest, includeLocation: Boolean): IepDetail {
```

Comment:
Renamed this because we also have `persistIepLevel` in the class and I was getting confused, happy to discuss